README.md

```
@@ -155,6 +155,16 @@ To enable the chat model to describe images, use the `enableVision()` method
 ```javascript
 chat.enableVision(true);
 ```
+### Add File to Gemini (optional)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)